### ISSUE TITLE
failed to start running the program

### ISSUE THREAD

**Issue by Yi-MingShi** [emzodls/tf_nn#3]
Hi Emzo,

Following the installation tutorial, I gave a try to run NeuRipp, but it showed up running errors as follow:

2020-02-11 17:16:56.077987: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-02-11 17:16:56.114869: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3696000000 Hz
2020-02-11 17:16:56.118005: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x7fffd56c04e0 executing computations on platform Host. Devices:
2020-02-11 17:16:56.118200: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>
WARNING:tensorflow:`implementation=0` has been deprecated, and now defaults to `implementation=1`.Please update your layer call.
WARNING:tensorflow:From /home/yimingshi/venv/neuripp/lib/python3.7/site-packages/tensorflow/python/keras/backend.py:4081: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.

My OS is a virtual machine of Ubuntu 18.04 LTS in Win10. I'm thinking if this could be the problem. I appreciate it if you could help.
